Sources/OpenAPIURLSession/URLSessionTransport.swift

```diff
@@ -230,10 +231,17 @@ var debugLoggingEnabled: Bool {
     get { _debugLoggingEnabled.withLockedValue { $0 } }
     set { _debugLoggingEnabled.withLockedValue { $0 = newValue } }
 }
-func debug(_ items: Any..., separator: String = " ", terminator: String = "\n") {
+private let _standardErrorLock = LockStorage.create(value: FileHandle.standardError)
```

Comment:
Where's the `LockStorage` type coming from?

Reply:
It's vendored in along with the vendored `AsyncBackpressuredStream`. If/when we come to replace it, we can find a suitable alternative for this.